data/json/items/tool/electronics.json

```diff
@@ -424,6 +424,79 @@
     ],
     "tick_action": [ "EPIC_MUSIC" ],
     "flags": [ "WATCH", "LIGHT_10", "TRADER_AVOID", "WATER_BREAK", "ELECTRONIC", "ALLOWS_REMOTE_USE" ]
+},
+  {
+    "id": "portable_dvd_player",
+    "type": "TOOL",
+    "category": "tools",
+    "name": { "str": "portable DVD player" },
+    "description": "A compact device with a 7-inch screen designed for playing both CDs and DVDs.  It's portable, making it convenient for travel or on-the-go entertainment in the Cataclysm.  This device also include simple earphones, built-in speakers, a rechargeable battery, and an USB and SD card slots for multimedia playback."
+    "symbol": ",",
+    "color": "dark_gray",
+    "material": [ "plastic", "aluminum" ],
+    "weight": "1350 g",
+    "volume": "1250 ml",
+    "longest_side": "17 cm",
+    "price": 75000,
+    "price_postapoc": 200,
+    "ammo": [ "battery" ],
+    "charges_per_use": 1,
+    "use_action": 
+    [
+      {
+        "target": "dvd_player_on",
+        "msg": "You turn the screen up and the device on.",
+        "menu_text": "Turn up the screen and turn on the device.",
+        "active": true,
+        "need_charges": 1,
+        "need_charges_msg": "The portable DVD player batteries need more charge.",
+        "type": "transform"
+      }
+    ],
+    "flags": [ "WATCH", "WATER_BREAK", "ELECTRONIC", "ALLOWS_REMOTE_USE" ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "rigid": true,
+        "flag_restriction": [ "BATTERY_MEDIUM" ],
+        "default_magazine": "medium_battery_cell"
+      },
+      {
+        "pocket_type": "CD/DVD entrance",
+        "rigid": true,
+        "max_contains_volume": "1 ml",
+        "max_contains_weight": "1 g",
+      }
+    ],
+    "melee_damage": { "bash": 2 }
+  },
+  {
+    "id": "dvd_player_on",
+    "copy-from": "portable_dvd_player",
+    "type": "TOOL",
+    "name": { "str": "portable DVD player - on", "str_pl": "portable DVD players - on" },
+    "longest_side": "34 cm",
+    "power_draw": "1 W",
+    "revert_to": "portable_dvd_player",
+    "use_action": [
+      "ELECTRICSTORAGE",
+      "EBOOKREAD",
+      {
+        "type": "effect_on_conditions",
+        "menu_text": "Watch a movie or listen to some music.",
+        "description": "Making sure you have a DVD or CD disc, watch a movie or listen to some music.",
+        "//need_charges_msg": "The portable DVD player batteries need more charge.",
+        "effect_on_conditions": [ "WATCH_MOVIE" ]
+      },
+      {
+        "ammo_scale": 0,
+        "target": "portable_dvd_player",
+        "msg": "You turn down the screen, also turning the device off.",
+        "menu_text": "Turn down the screen and turn off the device.",
```

Comment:
Same as before with the turn-on option. You could shorten it to something like, “Turn on.”

Reply:
Just made it.